activerecord/test/cases/tasks/sqlite_rake_test.rb

```diff
@@ -47,9 +50,11 @@ def test_db_create_when_file_exists
 
       def test_db_create_with_file_does_nothing
         File.stub(:exist?, true) do
-          ActiveRecord::Base.expects(:establish_connection).never
-
-          ActiveRecord::Tasks::DatabaseTasks.create @configuration, "/rails/root"
+          with_stubbed_method(ActiveRecord::Base, :establish_connection, nil) do
```

Comment:
Should be `assert_not_called(ActiveRecord::Base, :establish_connection) {}`

Reply:
Quite, fixing.